src/main/java/com/aviva/UseCases/Recommender.java

```diff
@@ -7,20 +7,30 @@
 import com.aviva.Entities.AccountHolder;
 import com.aviva.Entities.InputData;
 
+/**
+ * A use case class reposnible for handling the generating a client's recommended cars and inserting
+ * the cars into our database
+ */
 
 public class Recommender {
```

Comment:
If you think the Recommender class is a use case, then we should probably move it to CarRecommendations package because all of the other use cases are in there. Additionally, I think this class right now is acting as both a controller (since it is directing the input from the front end) and a use case (by interacting with the entities), which is violating clean architecture.